README.md

```diff
@@ -28,15 +33,55 @@ Or install it yourself as:
 $ gem install arclight
 ```
 
+Then, use Rails to generate the application:
+
+```sh
+$ rails generate arclight:install
+```
+
 ## Usage
 
-TODO: Write usage instructions here
+See the [ArcLight demo](https://arclight-demo.projectblacklight.org/) and [ArcLight MVP Wiki](https://github.com/sul-dlss/arclight/wiki) for usage.
+
+## Resources
+
+* General
+  * [ArcLight team wiki](https://wiki.duraspace.org/display/hydra/ArcLight): includes design process documentation
+  * Use the [ArcLight Google Group](http://groups.google.com/d/forum/arclight-community) to contact us with questions
+* MVP Implementation:
+  * [ArcLight demo site](https://arclight-demo.projectblacklight.org/)
+  * [YouTube channel](https://www.youtube.com/channel/UCbSaP93HdypsW6hXy7V1nFQ): MVP sprint demo videos
+  * [ArcLight MVP Wiki](https://github.com/sul-dlss/arclight/wiki)
+  * [Blacklight wiki](https://github.com/projectblacklight/blacklight/wiki)
+
+## ArcLight MVP Development team
+
+* Stanford University
+  * [Camille Villa](https://github.com/camillevilla)
+  * [Darren Hardy](https://github.com/drh-stanford)
+  * [Gary Geisler](https://github.com/ggeisler)
+  * [Jack Reed](https://github.com/mejackreed)
+  * [Jennifer Vine](https://github.com/jvine)
+  * [Jessie Keck](https://github.com/jkeck)
+  * [Mark Matienzo](https://github.com/anarchivist)
+* University of Michagan
+  * [Gordon Leacock](https://github.com/gordonleacock)
 
 ## Development
 
-After checking out the repo, run `bin/setup` to install dependencies. Then, run `bundle exec rake` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
+To install this gem onto your local machine, run
```

Comment:
@drh-stanford I think it might be unnecessary in our rails engine environment. It's going to create a packaged version of the gem and install that.  In our engine world, we instead reference the root of the engine in the `Gemfile` (using `path:`), so I'm not sure we make use of any packaged/installed gem for this.

Unless anybody else uses this as part of their development workflow, I think we may want to remove it.